docs/advanced_topics/settings.rst

```diff
@@ -268,6 +268,24 @@ This setting lets you override the maximum number of pixels an image can have. I
 This setting enables feature detection once OpenCV is installed, see all details on the :ref:`image_feature_detection` documentation.
 
 
+Documents
+---------
+
+.. code-block:: python
+
+  WAGTAILDOCS_SERVE_METHOD = 'redirect'
+
+Determines how document downloads will be served. Normally, requests for documents are sent through a Django view, to perform permission checks (see :ref:`image_document_permissions`) and potentially other housekeeping tasks such as hit counting. To fully protect against users bypassing this check, it needs to happen in the same request where the document is served; however, this incurs a performance hit as the document then needs to be served by the Django server. In particular, this cancels out much of the benefit of hosting documents on external storage, such as S3 or a CDN.
```

Comment:
Regarding the use of the phrase "will be served", this change alters not just the serving but also the linking. Could this say "will be linked to and served"?

More specifically, if someone chooses `'direct'`, it might not be super obvious that the _/documents/pk/filename_
URL pattern no longer appears across the site (even though, correctly, it still works, supporting, say, manual uses of `reverse` in other Django code). Specifically I think this means links both in rich text and in the documents admin views (list and edit) will point directly to the underlying storage.

Reply:
Updated in d09445eedffab911acc40e2d9567ffa7963b594e - I've slightly reworded the description of `direct` to make it clearer that we're no longer linking to the Django view, but open to other suggestions here.

Reply:
Looks great, thanks.